take into account that $TMPDIR may be a symlinked directory, this is required because path equality is checked
in some tests
author: Kenneth Hoste (HPC-UGent)
--- ipython-5.0.0/IPython/core/tests/test_paths.py.orig	2016-08-17 11:11:24.104380000 +0200
+++ ipython-5.0.0/IPython/core/tests/test_paths.py	2016-08-17 11:11:24.153308000 +0200
@@ -17,7 +17,7 @@
 from IPython.testing.decorators import skip_win32
 from IPython.utils.tempdir import TemporaryDirectory
 
-TMP_TEST_DIR = tempfile.mkdtemp()
+TMP_TEST_DIR = os.path.realpath(tempfile.mkdtemp())
 HOME_TEST_DIR = os.path.join(TMP_TEST_DIR, "home_test_dir")
 XDG_TEST_DIR = os.path.join(HOME_TEST_DIR, "xdg_test_dir")
 XDG_CACHE_DIR = os.path.join(HOME_TEST_DIR, "xdg_cache_dir")
